sdk-lib/src/main/java/cash/z/ecc/android/sdk/jni/RustBackendWelding.kt

```diff
@@ -54,21 +54,23 @@ internal interface RustBackendWelding {
 
     fun isValidUnifiedAddr(addr: String): Boolean
 
-    suspend fun getCurrentAddress(account: Int = 0): String
+    suspend fun getCurrentAddress(account: Account = Account.DEFAULT): String
```

Comment:
We could remove the default from here, because this is an internal API of the SDK and isn't public facing.  This is a good example where default arguments could cause problems.